web-wallet/src/lib/dusk/components/AnchorButton/AnchorButton.svelte

```diff
@@ -24,17 +24,21 @@
   export let size = "normal";
 
   /** @type {ButtonVariant} */
-  export let variant = "secondary";
+  export let variant = "primary";
 
   /** @type {String | Undefined} */
   export let text = undefined;
 
+  /** @type {Boolean} */
+  export let alt = false;
```

Comment:
Yes, right now, the only exception is the Settings button on the Dashboard layout.

I think these points are valid, we can refactor this so that the alt variant is only applied to this instance of the button.

Reply:
I think removing the `alt` prop and relying on contextual styling is the way to go.